targets/csharp/source/source/PluginManager.cs

```diff
@@ -25,7 +25,7 @@ private PluginManager()
         /// <param name="contract">The plugin contract.</param>
         /// <param name="instanceName">The optional plugin instance name. Instance names allow to have mulptiple plugins with the same contract.</param>
         /// <returns>The plugin instance.</returns>
-        public static T GetPlugin<T>(PluginContract contract, string instanceName = string.Empty) where T : IPlayFabPlugin
+        public static T GetPlugin<T>(PluginContract contract, string instanceName = "") where T : IPlayFabPlugin
```

Comment:
Consider using String.IsNullOrEmpty(s)
Not sure if we have one **s**tring.IsNullOrEmpty. If so, use that.